utils/print-deployments-logs.js

```diff
@@ -13,7 +13,7 @@ module.exports = async ({ github, context, fs }) => {
   const botCommentsArray = [];
 
   if (uniqueDeployUrl) {
-    defaultBody = `[Deployment: ${new Date()}](${uniqueDeployUrl})`;
+    defaultBody = `[Deployment: ${new Date()} | Commit: ${commitSha}](${uniqueDeployUrl})`;
```

Comment:
```suggestion
    defaultBody = `- [${commitSha}](${uniqueDeployUrl})`;
```

Can you also calculate the seven letter commit SHA like how GitHub displays it? I'm unsure if they slice the end or if they calculate it differently but it should match what shows in the UI (which if I recall is seven characters- sorry I'm on my phone GitHub client now it's not easy for me to check)

Reply:
Sure, I will update it